File: 001/001.py
#S = 4piR^2
#V = 4/3piR^3
#pi = 3.141593
from S_to_V import S_to_V

S = float(input("Nhap dien tich S: "))

V = S_to_V(S)

print(f"The tich V = {V}")

File: 001/001.js
const readline = require('readline');
const S_to_V = require('./S_to_V');

const rl = readline.createInterface({
	input: process.stdin,
	output: process.stdout
});

rl.question('Nhap dien tich S: ',(S) => {
	S = parseFloat(S);

	const V = S_to_V(S);
	console.log(`The tich V = ${V.toFixed(6)}`);

	rl.close();

})
